packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -64,7 +64,8 @@ export const BlockSwitcherDropdownMenu = ( { clientIds, blocks } ) => {
 				_icon = blockInformation?.icon; // Take into account active block variations.
 			} else {
 				const isSelectionOfSameType =
-					uniq( blocks.map( ( { name } ) => name ) ).length === 1;
+					[ ...new Set( blocks.map( ( { name } ) => name ) ) ]
```

Comment:
You could use [`Set.size`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/size) here.

Reply:
Good call, updated in https://github.com/WordPress/gutenberg/pull/43330/commits/ad02c59e4f568b6713f02c34f1f0a494e62f20d1